deep_water_level/src/deep_water_level/data.py

```diff
@@ -222,11 +266,13 @@ def get_data_loaders(
             train_dataset,
             batch_size=batch_size,
             shuffle=True,
+            drop_last=True,
         ),
         torch.utils.data.DataLoader(
             test_dataset,
             batch_size=batch_size,
             shuffle=False,
+            drop_last=True,
```

Comment:
what is drop_last, here and in the constructor below?

Reply:
If the last batch has size 1, the training breaks, because it's taken as a single element rather than a batch (not sure why exactly, but seemed like a Pytorch thing).   When you have big batches is unlikely to happen, but since I'm using a batch size of 2, I hit this issue.  Using this parameter would drop the last batch if it's not "full"